android/src/main/kotlin/de/ffuf/in_app_update/InAppUpdatePlugin.kt

```diff
@@ -214,6 +226,14 @@ class InAppUpdatePlugin : FlutterPlugin, MethodCallHandler,
         }
     }
 
+    private fun getFlexibleUpdateProgress(result: Result) = checkAppState(result) {
+        result.success(flexibleUpdateProgress)
```

Comment:
a stream would be nicer, but this should be good enough - thanks! Will merge soon